nin/frontend/app/scripts/services/camera.js

```diff
@@ -63,7 +63,7 @@
           ]) + ',';
         },
         getCameraLookat: function() {
-          var camera = layer.instance.camera;
+          if (!camera) return;
```

Comment:
Might as well use layer.instance.camera directly, in case it changes in the layer?